modules/node_modules/@ncigdc/containers/SmartSearchPage.js

```diff
@@ -94,6 +94,20 @@ class SmartSearchComponent extends React.Component {
           this.Error = null;
 
           this.setQuery();
+
+          document.querySelector('.btn-search-query').onclick = () => {
```

Comment:
do we need to worry about removing listener on destroy to avoid leaks (or multiple handlers being fired)?